smart_card_connector_app/src/window-main.js

```diff
@@ -59,8 +56,27 @@ GSC.ConnectorApp.Window.AppsDisplaying.initialize();
 GSC.ConnectorApp.Window.DevicesDisplaying.initialize();
 GSC.ConnectorApp.Window.LogsExporting.initialize();
 
+GSC.I18n.adjustElementsTranslation();
```

Comment:
Moved this line before showWindow call, so that the window is displayed when all titles are already in place (so that no flickering may occur).